src/modules/core/components/TokenEditDialog/TokenEditDialog.tsx

```diff
@@ -133,13 +135,61 @@ const TokenEditDialog = ({
           <Heading appearance={{ size: 'normal' }} text={MSG.noTokensText} />
         )}
       </DialogSection>
-      <DialogSection appearance={{ align: 'right' }}>
-        <Button
-          appearance={{ theme: 'primary', size: 'large' }}
-          text={MSG.buttonDone}
-          onClick={close}
-        />
-      </DialogSection>
+      <Form
+        initialValues={{
+          tokenAddress: '',
+          description: '',
+        }}
+        onSubmit={handleSubmit}
+        validationSchema={validationSchema}
+      >
+        {({
+          isSubmitting,
+          isValid,
+          dirty,
+          values,
+        }: FormikProps<FormValues>) => (
+          <>
+            <DialogSection>
+              <Paragraph className={styles.description}>
+                <FormattedMessage {...MSG.notListedToken} />
+              </Paragraph>
+              <TokenSelector
```

Comment:
We won't be using the "remove" token logic in the future:

![Screenshot from 2020-11-26 16-22-26](https://user-images.githubusercontent.com/1193222/100363575-98b68400-3005-11eb-9554-fed0d347222e.png)

So please remove it from the `TokenSelector`

_(Yes, I know, this is not really to do with this PR, but we still have to do it...)_

Reply:
@rdig done, please make sure to "recheck" last commit :D